app/src/features/projects/list/ProjectsListPage.tsx

```diff
@@ -150,7 +136,19 @@ const ProjectsListPage = () => {
 
   // Refresh projects when pagination or sort changes
   useEffect(() => {
-    refreshProjectsList();
+    const sort = firstOrNull(sortModel);
+    const pagination = {
+      limit: paginationModel.pageSize,
+      sort: sort?.field || undefined,
+      order: sort?.sort || undefined,
+
+      // API pagination pages begin at 1, but MUI DataGrid pagination begins at 0.
+      page: paginationModel.page + 1
+    };
+
+    projectsDataLoader.refresh(pagination, advancedFiltersModel);
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm sure there's a reason to disable the dependency warning here but it would be great to see a comment explaining why

Reply:
Done